src/App.tsx

```diff
@@ -19,13 +20,12 @@ const mainContainer = classnames(
 export default function () {
```

Comment:
Now let's name default exports in every project, so we can debug better in future